src/oc_erchef/apps/chef_db/src/chef_sql.erl

```diff
@@ -1004,7 +1004,7 @@ create_object(insert_cookbook_artifact_version = QueryName, Args) when is_list(A
         {conflict, _Why} = Conflict -> Conflict
     end;
 create_object(QueryName, Args) when is_atom(QueryName), is_list(Args) ->
-    sqerl:statement(QueryName, Args, count);
+    sqerl:select(QueryName, Args, count);
```

Comment:
Why did you need to change this to `select`?  Context for concern: creation of most other object types is done via insert statements, and this would affect them as well.

Reply:
We changed the insert statements to be a function in plpgsql, and those return in the form : {ok,[[{<<"add_user">>,<<>>}]]}
but select expects return in the form:  https://github.com/chef/sqerl/blob/master/src/sqerl.erl#L94-L101
which is why the change from statement to select.

Reply:
updated as discussed!